sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/ReactorReceiver.java

```diff
@@ -244,13 +267,31 @@ public Mono<Void> addCredits(int credits) {
         });
     }
 
+    @Override
+    public void addCredit(Supplier<Long> creditSupplier) {
+        assert this.isV2;
+        if (isDisposed()) {
+            throw new RejectedExecutionException("Cannot schedule credit flow when the link is disposed.");
+        }
+        try {
+            dispatcher.invoke(() -> {
```

Comment:
nit: consistency using `this.` or leaving it out. There are other instances but I didn't want to point them all out.  You can also just ignore `this.` suggestion. BAhahaha get it? :D

Reply:
I’m thinking of the below guideline for the issue, is this sounds good?

"Use _.this_ only to eliminate complier conflicts between class member variables and parameters of class methods (constructor, setter etc..), or there is a method local variable and the method has a lambda accessing class member variable with the same name. Even in the second case, readability likely to improve if method local variables uses a non-conflicting name"

Reply:
I did scan new v2 types and v2 changes and reduced the use of "this." per above guideline, now its mostly in Ctrs its used.